src/compas/robots/model/robot.py

```diff
@@ -178,6 +185,21 @@ def from_urdf_file(cls, file):
         urdf = URDF.from_file(file)
         return urdf.robot
 
+    def to_urdf_file(self, file, prettify=False):
```

Comment:
should we here do the same as what was discussed in relation to the file readers in general? just do `to_urdf` and differentiate between file or string based on what is passed as parameter to the function?

Reply:
I thought we discussed something a little different from this.  That `read` should take either a file-like object or a file path.  I guess one could argue that `to_urdf_file` should take a file path or a file-like object and write to those places, but the `to_string` methods are different. They return a string.  I don't think the two should be combined.

Reply:
sounds good